app/cmd/rest.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func StartRest() {
+	controller()
+}
+
+func controller() {
+	http.HandleFunc("/health", health)
+	http.HandleFunc("/hello-world", helloWorld)
+
+	fmt.Print("Starting server on port 3000\n")
+	err := http.ListenAndServe(":3000", nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func health(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" && r.URL.Path != "health" {
+		errorHandler(w, r, http.StatusNotFound)
+		return
+	}
+
+	type response struct {
+		Status string `json:"status"`
+	}
+
+	data := response{"healthy"}
+
+	w.WriteHeader(http.StatusOK)
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
if you set header after`WriteHeader`, 
if wont have any effect, you can check your app header response to validate,
you might want to write it like this, also no need to explicitly write header for `200`

```suggestion
	w.Header().Set("Content-Type", "application/json")
	w.WriteHeader(http.StatusOK)
```

Reply:
I see, when I test my current condition. it didn't return the header as an `application/json` Thank you sensei 👍